src/consumption.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+ALPHA = 0.1727
+BETA =  0.219
+
+
+def fuel_consumption(velocity: np.array,
+                     alpha: float=ALPHA, beta: float=BETA):
+    """
+    Params
+    ------
+    velocity : numpy.array
+        En nudos
```

Comment:
Si `velocity` es un array, enonces `fuel` será un array. Si es así, no sé qué sentido tendría. Si asumes que la velocidad es ya escalar (supongo que es el módulo), puede dar lugar a confusión que lo declares como array.

```suggestion
    velocity : float
        En nudos
```

Reply:
Cierto, velocity es un escalar (el módulo). Me gustaría reconvertir esta función para que tome un array con velocidades y te compute el consumo de cada una.